subprojects/tooling-api/eclipse.gradle

```diff
@@ -0,0 +1,10 @@
+eclipse {
```

Comment:
:x: should stay inlined in the `.gradle.kts` script, similar logic exists in `IdePlugin` for inspiration

Reply:
I included the script.